### ISSUE TITLE
Small avatars are blurry

### ISSUE THREAD

**Issue by Jerome-Herbinet** [nextcloud/mail#10026]
### Steps to reproduce

I'm a bit afraid of changing things in nextcloud-vue (so I only open this issue for the moment) but if you check my test bellow, the CSS changes can fix avatar when they are too small (avoids blurry effect) : 

### CSS changes
![Capture d’écran du 2024-08-20 14-20-16](https://github.com/user-attachments/assets/985cde99-2b6a-4bd4-be02-7251d65117d5)

### Before 
![Capture d’écran du 2024-08-20 13-59-31](https://github.com/user-attachments/assets/89d632a0-2471-4641-acf0-71d34c511ff8)

### After 
![Capture d’écran du 2024-08-20 14-00-20](https://github.com/user-attachments/assets/000e4d9c-0295-42ae-80d8-9085aa6f8acf)

### Expected behavior

Read above

### Actual behavior

Read above

### Mail app version

3.7.7

### Operating system

CentOS

### PHP engine version

PHP 8.2

### Web server

Nginx

### Database

MariaDB

**Comment by Jerome-Herbinet** [comment]
@nextcloud/designers

**Comment by ChristophWurst** [comment]
Those are favicons. The favicon probably only has a low res icon.

**Comment by Jerome-Herbinet** [comment]
> Those are favicons. The favicon probably only has a low res icon.

@ChristophWurst you're right, we're talking about favicons, and it's not always available at all resolutions, including the highest ones. Do you confirm that my suggestion is relevant ?

**Comment by ChristophWurst** [comment]
I can confirm that low res favicons do not look great but I'll leave the decision on a workaround to designers

**Comment by jancborchardt** [comment]
@Jerome-Herbinet could you try with a few other favicons, including those which have higher resolution? A thing I would be worried about is that it will result in small squares inside the circle, which looks off.

In your example it looks good cause it’s a favicon with white background in light mode, but this would look differently otherwise. So I am inclined to reject this and suggest to improve the favicon instead?

**Comment by jancborchardt** [comment]
Otherwise, if we know that a favicon is low-res, we could have special handling for it. But only for the low-res ones, cause high-res ones pose no issue.

**Comment by Jerome-Herbinet** [comment]
> @Jerome-Herbinet could you try with a few other favicons, including those which have higher resolution? A thing I would be worried about is that it will result in small squares inside the circle, which looks off.
> 
> In your example it looks good cause it’s a favicon with white background in light mode, but this would look differently otherwise. So I am inclined to reject this and suggest to improve the favicon instead?

@jancborchardt : test with a bigger square one
![2024-08-21_08-55](https://github.com/user-attachments/assets/692c5723-0bea-4631-9b03-f4659be6d018)

**Comment by Jerome-Herbinet** [comment]
> @Jerome-Herbinet could you try with a few other favicons, including those which have higher resolution? A thing I would be worried about is that it will result in small squares inside the circle, which looks off.
> 
> In your example it looks good cause it’s a favicon with white background in light mode, but this would look differently otherwise. So I am inclined to reject this and suggest to improve the favicon instead?

@jancborchardt : test with a bigger square one
![2024-08-21_08-58](https://github.com/user-attachments/assets/d5a79b5f-92f7-4fa0-b865-c695a498f270)

**Comment by jancborchardt** [comment]
Right, so your solution would basically ensure that lower-res images don’t get upscaled – that is good.

As said one thing I am worried about is it looking weird with low-res favicons of different colors, as it would result in small squares among the proper circles (or rounded squares, depending on how the favicon looks). So it would be good to see some examples with that as well. Also for example in your screenshot, the Nextcloud logo in the circle is smaller than the personal picture?

**Comment by Jerome-Herbinet** [comment]
> Right, so your solution would basically ensure that lower-res images don’t get upscaled – that is good.
> 
> As said one thing I am worried about is it looking weird with low-res favicons of different colors, as it would result in small squares among the proper circles (or rounded squares, depending on how the favicon looks). So it would be good to see some examples with that as well. Also for example in your screenshot, the Nextcloud logo in the circle is smaller than the personal picture?

@jancborchardt : test with a smaller square one

![2024-08-21_11-29](https://github.com/user-attachments/assets/0bb0d059-5394-4bcb-8bef-cb2a82095353)

**Comment by jancborchardt** [comment]
@Jerome-Herbinet the examples are always very isolated. :) It’s best to show a whole list of mails with various avatars and favicons, so it can be judged whether it works fine in practice.

I do like that regardless of size they get rounded, I’m just not sure it will look nice with various circle sizes in the list. But yeah, the question is if the varying circle sizes are nicer than the pixelation.

**Comment by jancborchardt** [comment]
@Jerome-Herbinet I'm fine either way, and I prefer if design is not decided by polls (especially small things like this). I'm just asking for the screenshot to show further different examples in context, so we can see how it looks in practice. :)

**Comment by ChristophWurst** [comment]
Dear @Jerome-Herbinet @jancborchardt,
how can we proceed with this? Could one of you create example screenshots to find consent?
Thank you

**Comment by jancborchardt** [comment]
To not hold it up further, I'd also be happy to +1 a pull request that implements @Jerome-Herbinet's changes as he wrote. :) as said it should be fine.
